include/Metabox/Posts/Tagtocategory.php

```diff
@@ -0,0 +1,110 @@
+<?php
```

Comment:
The filename should be `TagToCategory`. Class name should be `BM_Metabox_Posts_TagToCategory`

Reply:
Fixed